docs/xls-33d-multi-purpose-tokens/multi-purpose-tokens.md

```diff
@@ -0,0 +1,60 @@
+
+They reduce the long-term infrastructure and storage burdens for node operators, increasing network resiliency.
+
+MPTs also improve node perfomance when processing large volumes of transactions.
+
+MPTs are unidirectional. While trust lines use "balance netting," MPTs have only a single balance.
+
+An account can issue a maximum of 32 unique MPT issuances. If an issuer wants to support more than this number of MPTs, they can open additional accounts.
+
+Since token holders will not acquire an MPT without first making an off-ledger trust decision, MPTs have no trust limits. For example, a common use case for an MPT is a fiat-backed stablecoin, where a token holder wouldn't purchase more stablecoins than they would feel comfortable holding.
+
+Unlike some existing capabilities of the ledger, MPTs are not subject to rippling, and  do not require configurability settings related to that functionality.
+
+## MPTs versus IOUs
+
+On a technical level, MPTs provide a fundamentally different way to represent fungible tokens on the ledger.  While IOUs are represented by trustlines and have bilateral debt relationships, MPTs use a simpler, unilateral relationship captured by an MPToken object. This results in substantial space savings on the ledger. The representation of a fungible token as a token object instead of a trustline makes it easier to enable functionality for real-world financial assets on-chain, such as token-level metadata, fixed supply, and fixed-point balance.
+
+On a usage level, MPTs provide a straightforward conceptual model compared to trustlines and rippling. Developers can more easily build web3 applications around `MPToken` and `MPTokenIssuance` objects, with some similarities to the conceptual model of XLS-20 NFTs.  It is also simpler for ordinary users to understand what tokens are available, what tokens they have issued, and what they hold in their wallet.  For both issuers and holders of MPTs, there will typically be a smaller XRP reserve compared to the equivalent representations with IOU trustlines.
+
+MPTs are intended to be complementary to IOUs.  While there may be use cases where either MPTs and IOUs might be suitable, we foresee a need for both over the long term.  There will be use cases such as credit lines for lending and borrowing that might be better represented by IOUs long term.  As the MPT feature set should evolve in an incremental manner to unlock more common use cases first and deliver other feature support at a later time. During the MPT development period, some cases might still be better represented by an IOU, then later be better supported with MPTs.
+
+## Hexadecimal Values
+
+Several of the amount fields used with MPTs are stored as string representations of hexadecimal values (for example, `MaximumAmount`, `OutstandingAmount`, `MPTokenAmount`). The following hexadecimal value represents the decimal value 1000.
```

Comment:
We should clarify that all of these fields have the internal type of `Uint64`. 

In fact, one could make a general statement that any fields with internal type of `Uint64` are represented in hexadecimal strings in JSON.

An updated example would be: 
> Several of the amount fields _**(which have the internal type of `Uint64`)**_ used with MPTs are stored as string representations of hexadecimal values in _**JSON**_ (for example, `MaximumAmount`, `OutstandingAmount`, `MPTokenAmount`).

Feel free to change it however you like.